tests/custom-behavior.test.js

```diff
@@ -71,6 +71,41 @@ test("test mixed custom behavior sources", async () => {
   ).toBe(true);
 });
 
+test("test custom behaviors from git repo", async () => {
+  const res = child_process.execSync(
+    "docker run -v $PWD/test-crawls:/crawls webrecorder/browsertrix-crawler crawl --url https://specs.webrecorder.net/ --url https://example.org/ --url https://old.webrecorder.net/ --customBehaviors \"git+https://github.com/webrecorder/browsertrix-crawler.git?branch=custom-behaviors-testing&path=tests/custom-behaviors\" --scopeType page",
```

Comment:
I guess we should change this to main for merge, so it uses this branch itself?

Reply:
We can switch it to main, but I also created the referenced `custom-behaviors-testing` branch so as long as we don't delete that it should continue to be okay.

Reply:
Changed to use main to make it less brittle